templates/default/flinkhistoryserver.service.erb

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description = Flink historyserver
+After = syslog.target network.target remote-fs.target <%= @deps %>
+
+[Service]
+User = <%= node['hops']['hdfs']['user'] %>
```

Comment:
Why do we need to run the flinkhistoryserver as `hdfs`?

Reply:
We don't any more, will change it to `flink` user